sdk/compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Util.hs

```diff
@@ -459,6 +459,39 @@ instance Show UpgradingDep where
       Just udPackageVersion -> " (v" <> show udPackageVersion <> ")"
       Nothing -> mempty
 
+------------------------------------------------------------------------
+-- Shorthands
+------------------------------------------------------------------------
+
+-- Types
+tvar :: T.Text -> Type
+tvar = TVar . TypeVarName
+
+tunit :: Type
+tunit = TBuiltin BTUnit
+
+tint :: Type
+tint = TBuiltin BTInt64
+
+tbool :: Type
+tbool = TBuiltin BTBool
+
+tyLamTyp :: Type
+tyLamTyp = TForall (a, typToTyp) (TVar a :-> TVar a)
+  where
+    a = TypeVarName "a"
+    typToTyp = KArrow KStar KStar
+
+tcon :: T.Text -> Type
+tcon t = TCon $ Qualified SelfPackageId (ModuleName ["Main"]) (TypeConName [t])
```

Comment:
As long as we're defaulting the module name to Main here, I think it's probably best if we rename this identifier (and other testing-only functions such as `mkEmptyModule`) to have a "Test" suffix, to make it clear that it should only be used in tests. Otherwise we run the risk of using this during development of an actual feature, where it doesn't make any sense to default the module name.

Alternatives you can go for if you prefer:
- move this to the encode/decode tests
- move this (and other testing-only functions such as `mkEmptyModule`) to a different, "Test-only utils" module